crypto/rand/rand_win.c

```diff
@@ -34,52 +34,44 @@
 #  define INTEL_DEF_PROV L"Intel Hardware Cryptographic Service Provider"
 # endif
 
-static void readtimer(void);
-
 int RAND_poll(void)
 {
-    MEMORYSTATUS mst;
 # ifndef RAND_WINDOWS_USE_BCRYPT
     HCRYPTPROV hProvider;
 # endif
     DWORD w;
     BYTE buf[64];
```

Comment:
512 bits per poll call?  This seems high.  128 (16 bytes) bits should be fine for stirring things.